### PR TITLE
Update to source location in README.md file

### PR DESCRIPTION
**Pull request by mziyabo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ ODataPy is an open-source Python library that implements the Open Data Protocol
 
 ## Getting the source
 
-    git clone https://odatateam.visualstudio.com/DefaultCollection/_git/odatapy
+    git clone https://github.com/OData/odatapy-client.git
 
 ## Getting ODataCpp and building
```
Comment (chinadragon0515) [inline README.md:10]: Can you help to update ODataCpp to ODataPy? thanks

Comment (mziyabo) [inline README.md:10]: Sure. Is this update on line 10? Because that part looks fine as is.

Comment (chinadragon0515) [inline README.md:10]: Line 10,11,12, three line, thanks for your great help.

Comment (mziyabo) [inline README.md:10]: Those lines are ok because we need the ODataCpp sources for ODataPy. No further change is required there. Thanks.

Comment (chinadragon0515) [inline README.md:10]: thanks, merged.

**Pull request merged**